exports.go

```diff
@@ -45,6 +47,28 @@ func (e *Export) Validate(vr *ValidationResults) {
 	e.Subject.Validate(vr)
 }
 
+// Revoke enters a revocation by publickey and timestamp into this export
+// If there is already a revocation for this public key that is newer, it is kept.
+func (e *Export) Revoke(pubKey string, timestamp time.Time) {
```

Comment:
Same comment, since time will mostly be time.Now, make simple Revoke() not take a time. We can add in one that sets it if needed.